source/docs/docs/identity-and-access-management/tenancy.md

```diff
@@ -0,0 +1,67 @@
+# Tenancy
+
+## Scope
+
+Tenancy is the concept of providing isolated access to resources. In the context of the Infratographer ecosystem, we are talking about the isolation of data and the ability to provide a single application to multiple tenants. The scope of this document is to provide a shared understanding and a working model for developing multi-tenant applications within the Infratographer ecosystem. It is outside the scope to discuss multi-tenancy in other systems beyond Infratographer except where drawing comparison helps to clarify our intent.
+
+## Principles of Tenancy
+
+### Tenant Trees are Isolated
+
+Tenants provide isolated access to data. Access to data within a tenant tree needs to be explicitly granted.
+
+#### Rationale
+
+Tenant isolation limits access privileges to data and control over resources managed by the ecosystem.
+
+#### Implications
+
+Access to a parent tenant is not implied by granting access to a child tenant and access to one tenant does not imply access to other unrelated tenant trees. [Access granted to a parent tenant *does* imply granting access to child tenants](#tenant-permissions-are-inherited)
+
+### Tenant permissions are inherited
+
+Access granted on a tenant will be inherited by all subtenants.
+
+#### Rationale
+
+Inheriting access rights from the parent is more intuitive and fits better into existing authorization paradigms.
+
+#### Implication
+
+When access is granted to a tenant, that access will be granted to all subtenants of the tenant at the same level. Access cannot be limited to only one level unless the tenant has no children. In some cases, a flat tenant structure may be better serve isolation requirements.
+
+### Tenants are Hierarchical
+
+Tenants form a tree structure, and are not limited to a single level.
+
+#### Rationale
+
+Experience with multiple large projects (Kubernetes, Openstack, etc) has shown that many ecosystems start with single level tenancy and then migrate to hierarchical tenancy in the future. This migration is often fraught with peril, bugs and dragons.
```

Comment:
If it helps the case, GCP has quite a transparent view of the hierarchical model they offer [1], which was also one of the references in the initial multi-tenancy doc in fertilesoil.

[1] https://cloud.google.com/resource-manager/docs/cloud-platform-resource-hierarchy